doc/releases/release-notes-2.5.rst

```diff
@@ -196,6 +196,27 @@ Drivers and Sensors
 
 * CAN
 
+  * We reworked the configuration API.
+    A user can now either specify the timing manually
+    (define prop segment, phase segment1, phase segment2 and prescaler),
+    or use a newly introduced algorithm to calculate optimal timing values from
+    a bitrate and sampling point. The bitrate and sample point can be specifier
+    in the device tree too. It is possible to change the timing values
+    at runtime now.
+
+  * zcan_frame struct was reworked due to undefined behavior.
+    The std_id (11-bit) and ext_id (29-bit) are merged to a single id
+    field (29-bit). The union of both IDs was removed.
+
+  * We made the CANbus API was made CAN-FD compatible.
```

Comment:
```suggestion
  * The CAN bus API was made CAN-FD compatible.
```

Reply:
Passive voice should be avoided.

Reply:
Really? All the other entries are passive and I also felt it was a bit personal for a changelog. Your call though, the content itself isn't affected :-)

Reply:
At least Grammarly told me that after I pasted it there :-)
Don't you think it's better to read? The passive voice feels boring, IMO.